### PR TITLE
Project initialization

### PR DESCRIPTION
**Pull request by mcovalt**
This PR brings in the [Girder vue-project-template](https://github.com/girder/vue-project-template).

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: CI
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
```
Comment (naglepuff) [inline .github/workflows/main.yml]: Should these be `main`?

**Review by naglepuff — CHANGES_REQUESTED** [review]
Just a quick question about the CI config.

File: .github/workflows/main.yml (hunk 1 above)
Comment (mcovalt) [reply .github/workflows/main.yml]: Nice catch. Thanks!

Comment (mcovalt) [reply .github/workflows/main.yml]: Resolved in 904e500a41aa1bdee02549b5ef42a38a0dc83066.

**Review by naglepuff — APPROVED** [review]

**Pull request merged**